service/db/src/main/java/gov/va/vro/service/db/mapper/ExamOrderInfoResponseMapper.java

```diff
@@ -0,0 +1,37 @@
+package gov.va.vro.service.db.mapper;
+
+import gov.va.vro.model.claimmetrics.response.ExamOrderInfoResponse;
+import gov.va.vro.persistence.model.ExamOrderEntity;
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.ReportingPolicy;
+
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.util.List;
+
+@Mapper(
+    componentModel = "spring",
+    unmappedTargetPolicy = ReportingPolicy.IGNORE,
+    uses = ContentionInfoMapper.class,
+    injectionStrategy = InjectionStrategy.CONSTRUCTOR)
+public interface ExamOrderInfoResponseMapper {
+
+  @Mapping(
+      target = "hasAssociatedClaimSubmission",
+      expression = "java(examOrderEntity.getClaimSubmission() != null)")
+  @Mapping(
+      target = "orderedAt",
+      expression = "java(convertOrderedAt(examOrderEntity.getOrderedAt()))")
+  ExamOrderInfoResponse toExamOrderInfoResponse(ExamOrderEntity examOrderEntity);
+
+  List<ExamOrderInfoResponse> toExamOrderInfoResponses(Iterable<ExamOrderEntity> examOrderEntities);
+
+  default LocalDateTime convertOrderedAt(OffsetDateTime offsetOrdered) {
```

Comment:
I am wondering if MapStruct actually does this conversion automatically, especially since OffsetDateTime has its own built-in `toLocalDateTime()` method

If so this might kill the need to declare this default method and we would be able to just have a flat declaration like
```Java
@Mapping(
  target = "orderedAt",
  source = "orderedAt")
```
instead....

There's something about String-based method call expressions in Java that doesn't sit right with me. But if we must, we must.

Reply:
So digging into the map struct documentation it can do quite a number of translations automatically around primatives, their corresponding class types, and things to strings.

The list of date compatibilities is long but unfortunately it looks to be centered more around older joda and java 7 dates to local date time in java 8, with nothing listed for offset date time. That leads me to believe it wont work, although we could verify by testing, that's a fair chunk of day to remove one line of code on an off chance.